apidef/api_definitions.go

```diff
@@ -908,9 +908,19 @@ type UptimeTests struct {
 }
 
 type UptimeTestsConfig struct {
-	ExpireUptimeAnalyticsAfter int64                         `bson:"expire_utime_after" json:"expire_utime_after"` // must have an expireAt TTL index set (http://docs.mongodb.org/manual/tutorial/expire-data/)
-	ServiceDiscovery           ServiceDiscoveryConfiguration `bson:"service_discovery" json:"service_discovery"`
-	RecheckWait                int                           `bson:"recheck_wait" json:"recheck_wait"`
+	// ExpireAnalyticsAfter controls the expire time in seconds.
```

Comment:
why analytics? are these recorded in analytics and then we want to remove the entries from analytics?